tests/recipes/cad1/task1/test_evaluate.py

```diff
@@ -0,0 +1,107 @@
+import os
+import tempfile
+
+import numpy as np
+import pytest
+
+from recipes.cad1.task1.baseline.evaluate import (
+    ResultsFile,
+    compute_haaqi,
+    make_song_listener_list,
+    set_song_seed,
+)
+
+fs = 16000
+audiogram_freq = np.array([250, 500, 1000, 2000, 4000, 6000])
+audiogram = np.array([10, 20, 30, 40, 50, 60])
+audiogram_frequencies = np.array([125, 250, 500, 1000, 2000, 4000, 8000])
+
+
+def test_ResultsFile():
+    with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
`pytest` has a `tmp_path` you can use, see [here](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html). Its a pathlib `Path` object

You'd instead write...

```python
def test_ResultsFile(tmp_path):
    results_file = tmp_path / "results.csv"
    ...
```